File: Keanu.cpp
#include <bits/stdc++.h>
using namespace std;
#define ll long long int

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    int n;
    cin>>n;
    if(n%2==0){
        cout<<((n*n)/2)<<" casas brancas e "<<(n*n)/2<<" casas pretas\n";
    }
    else{
        cout<<((n*n)/2)+1<<" casas brancas e "<<(n*n)/2<<" casas pretas\n";


    }
    return 0;
}

File: Keanu.py
a = int(input())
b = a * a

if a % 2 == 0:
    # যদি n জোড় হয়
    res = b // 2
    print(f"{res} casas brancas e {res} casas pretas")
else:
    # যদি n বিজোড় হয়
    white = (b // 2) + 1
    black = b // 2
    print(f"{white} casas brancas e {black} casas pretas")